acceptance/tests/connect/connect_proxy_lifecycle_test.go

```diff
@@ -138,7 +139,8 @@ func TestConnectInject_ProxyLifecycleShutdown(t *testing.T) {
 			require.Len(t, pods.Items, 1)
 			clientPodName := pods.Items[0].Name
 
-			var terminationGracePeriod int64 = 60
+			// We should terminate the pods shortly after envoy gracefully shuts down in our 15s test cases.
+			var terminationGracePeriod int64 = 16
```

Comment:
Maybe we could potentially make this 5 seconds and make the 60 second timeout below something like 30s instead. No worries if that may be more unstable/flaky though

Reply:
Yeah, I like that. I'll also change the grace period in the tests to 5s.